modules/custom/az_dam/config/quickstart/core.entity_form_display.media.ib_dam_embed.default.yml

```diff
@@ -0,0 +1,62 @@
+uuid: 3e3c5ab9-8755-422a-a3db-661b855c0d2a
```

Comment:
We omit `uuid`s from exported config to make the config reusable across multiple sites.
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.